src/components/ParameterRow.js

```diff
@@ -178,8 +178,8 @@ export default class ParameterRow extends Component {
   }
 
   // Kong change - format error
-  getParameterRowErrorProps = (errors) => {
-    return errors?.toJS ? errors.toJS() : []
+  getParameterRowErrorProps = (errors, param) => {
+    return errors?.toJS ? errors.toJS().map(err => `Invalid value for property ${param.name} in ${param.in} section: ${err}`) : []
```

Comment:
But in that case when we have multiple errors the text ``Invalid value for property ${param.name} in ${param.in} section: ${err}`` will be repeated for each error? Am I right? Wouldn't be better to add another error message to that array then?

Reply:
Hmmmm interesting, yes. So it could be something like:

"Invalid value for property ${param.name}:" and then all of the errors joined by ","?